examples/detail_servers.py

```diff
@@ -0,0 +1,53 @@
+"""Fetch servers fixes details by CVE"""
+
+import os
+import csv
+import logging
+from configparser import ConfigParser
+from cbw_api_toolbox.cbw_api import CBWApi
+
+CONF = ConfigParser()
+CONF.read(os.path.join(os.path.abspath(os.path.dirname(__file__)), '..', 'api.conf'))
+CLIENT = CBWApi(CONF.get('cyberwatch', 'url'), CONF.get('cyberwatch', 'api_key'), CONF.get('cyberwatch', 'secret_key'))
+
+def to_csv_lines(cve_catalog):
+    """Generates a list of dictionnary objects"""
+    csv_lines = []
+    index_cve = 0
+    size_catalog = len(list(cve_catalog))
+    for cve in cve_catalog:
+        cve_details = CLIENT.cve_announcement(cve.cve_code)
+        index_cve += 1
+        print("Progress : {}/{} -- {}".format(index_cve, size_catalog, cve.cve_code), end="\r")
+        cve_servers = cve_details.servers
+        for server in cve_servers:
+            server_update = server['server'].updates
+            server_id = server['server'].id
+            if len(server_update) > 0:
+                csv_lines.append({'id server': server_id, 'CVE code': cve.cve_code,
+                                  'vendor': server_update[0]['target']['vendor'],
```

Comment:
We don't handle the current part of the update, which is present only for non KB updates. We should have a colum to display it. This column would be empty for KBs